### DISCUSSION TITLE
what is the security risk of enlabing direct access by ip if pw strong ?

### DISCUSSION THREAD

**Discussion by nykoo** [rustdesk/rustdesk#1214] [General]
Hello,

What is the security risk of enlabing direct access by ip if a strong password is set to rustdesk access on computers ?

I'm not able to set my own server (i tried but i really not have the skills for) so i would like to enabled access by ip on some distant computers but would like to know what is the risk.

Thank you for your answers :)

**Comment by bigben446** [comment]
密码再加一个证书，基本上应该没有问题了

**Comment by nykoo** [comment]
i ask beacause of this comment from @rustdesk 

https://github.com/rustdesk/rustdesk/issues/390#issuecomment-1045938738

**Reply by DarkSoliditi** [reply]
The Issue mentioned in the comment you referenced doesn't have anything to do with rustdesk, It's generally a bad idea to make your computer a static location on the internet, that's the security issue they're talking about and its one that you only have to worry about if you have have a External Static IP, and if you don't know what a External Static IP is, then you have nothing to worry about.
